terraform/deployments/cluster-services/aws_ebs_csi_driver.tf

```diff
@@ -0,0 +1,33 @@
+resource "helm_release" "csi_driver" {
+  chart            = "aws-ebs-csi-driver"
+  name             = "aws-ebs-csi-driver"
+  namespace        = "kube-system"
+  repository       = "https://kubernetes-sigs.github.io/aws-ebs-csi-driver"
+  version          = "1.6.2" # TODO: Dependabot or equivalent so this doesn't get neglected.
```

Comment:
The [current version of this chart](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/Chart.yaml#L5) appears to be `2.6.8`. (1.6.2 is the app version.)